deploy-as-code/helm/charts/core-services/service-request/values.yaml

```diff
@@ -0,0 +1,64 @@
+# Common Labels
+labels:
+  app: "service-request"
+  group: "core"
+
+# Init Containers Configs
+initContainers:
+  dbMigration:
+    enabled: true
+    schemaTable: "service-request_schema"
```

Comment:
change it to service_request_schema as all history tables follow the same naming convention